### PR TITLE
Atlantis e2e tests for policy checks step

### PR DESCRIPTION
**Pull request by msarvar**

### CODE REVIEW

File: server/events/approve_policies_command_runner.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 func NewApprovePoliciesCommandRunner(
 	commitStatusUpdater CommitStatusUpdater,
 	prjCommandBuilder ProjectApprovePoliciesCommandBuilder,
-	prjCommandRunner ProjectPolicyCheckCommandRunner,
+	prjCommandRunner ProjectApprovePoliciesCommandRunner,
```
Comment (nishkrishnan) [inline server/events/approve_policies_command_runner.go:12]: Adding this change here is going to make it tricky to upstream this as the refactor only exists in our fork.   Maybe I should just upstream the refactor

**Review by nishkrishnan — APPROVED** [review]
LGTM

File: server/events/approve_policies_command_runner.go (hunk 1 above)
Comment (msarvar) [reply server/events/approve_policies_command_runner.go:12]: It should be trivial to fix it upstream. I will update it, also your refactor builds on top of my policy_check PR that had some refactoring already done. We can potentially include your refactor in my PR.

**Pull request merged**